examples/permissioned-voting/README.md

```diff
@@ -0,0 +1,39 @@
+# Example Permissioned Voting Stateful Smart Contract Application
+
+This project demonstrates how to create a Permissioned Voting Stateful Smart Contract Application.
+To create this type of application on Algorand, there are four steps that must be supported:
+
```

Comment:
Please add a link to the original article presenting voting smart contract.

Reply:
ok